tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/GraphQL/HotChocolate/HotChocolateCommon.cs

```diff
@@ -45,6 +45,45 @@ internal static Scope CreateScopeFromExecuteAsync<T>(Tracer tracer, in T request
             return scope;
         }
 
+        internal static void UpdateScopeFromExecuteAsyncV13<TOperationContext>(Tracer tracer, in TOperationContext context)
+        where TOperationContext : IOperationContextV13
+        {
+            if (!tracer.Settings.IsIntegrationEnabled(IntegrationId))
+            {
+                // integration disabled, don't create a scope, skip this trace
+                return;
+            }
+
+            var scope = tracer.InternalActiveScope;
+            var span = scope?.Span;
+            if (span == null || span.OperationName != ExecuteOperationName)
+            {
+                // not in a Hotchocolate execution span
+                return;
+            }
+
+            try
+            {
+                var operation = context.Operation;
+                var operationType = operation.OperationType.ToString();
+                if (span.Tags is GraphQLTags tags)
+                {
+                    span.ResourceName = $"{operationType} {tags.OperationName ?? "operation"}";
+                    tags.OperationType = operationType;
+                }
+                else
+                {
+                    var operationName = span.GetTag(Trace.Tags.GraphQLOperationName);
```

Comment:
The operation name is always empty. We have the same issue with older version. If you want to go the extra mile, would be great to check why it is so and if we can do something about it